### PR TITLE
[SG-1]: create login/signup screen

### PR DESCRIPTION
**Pull request by zuhair5**
## jira
[TSG-1](https://shopgoswift.atlassian.net/browse/SG-1)

Description:
- create a login/signup screen
- add buttons using the same UI as in the screenshot
- complete auto layout

how to test ;
- navigate using the main view controller to the appropriate view controller

screenshots:
Login:Signup Screen

### CODE REVIEW

File: ShopGo/LoginScreen/LoginSignupVC.swift
Comment (aliahmed1105) [inline ShopGo/LoginScreen/LoginSignupVC.swift:1]: Rename the view controller to LoginSignupViewController.

File: ShopGo/LoginScreen/LoginSignupExtenstion.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  ShopGo
+//
+//  Created by Quratulain on 7/26/23.
+//
+
+import UIKit
+
+extension LoginSignupVC {
+    // function for radius of login button inside the view
+    func loginButtonRadius() {
```
Comment (aliahmed1105) [inline ShopGo/LoginScreen/LoginSignupExtenstion.swift]: In the future we can do much more setup in the function besides radius so rename this to setupLoginButton().

**Review by aliahmed1105 — CHANGES_REQUESTED** [review]

File: ShopGo/LoginScreen/LoginSignupExtenstion.swift (hunk 1 above)
Comment (zuhair5) [reply ShopGo/LoginScreen/LoginSignupExtenstion.swift]: already created a custom class for it, this function is only going to be used for the login button.

File: ShopGo/LoginScreen/LoginSignupVC.swift
Comment (zuhair5) [reply ShopGo/LoginScreen/LoginSignupVC.swift:1]: fixed

**Review by aliahmed1105 — APPROVED** [review]

File: ShopGo/LoginScreen/LoginSignupExtenstion.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+//  Created by Quratulain on 7/26/23.
+//
+
+import UIKit
+
+extension LoginSignupViewController {
+    // function for radius of login button inside the view
+    func buttonCornerStyle() {
+        loginButtonOutlet.layer.cornerRadius = 22
+        loginButtonOutlet.layer.masksToBounds = true
```
Comment (haroonmaqsoodch) [inline ShopGo/LoginScreen/LoginSignupExtenstion.swift]: do we need this method? We already have similar stuff in `CustomButton.swift` can we not reuse that?

Comment (zuhair5) [reply ShopGo/LoginScreen/LoginSignupExtenstion.swift]: I'll create a separate class for this too

Comment (zuhair5) [reply ShopGo/LoginScreen/LoginSignupExtenstion.swift]: fixed

**Comment by haroonmaqsoodch** [comment]
another pr has been raised

**Pull request closed without merging**